packages/string-templates/src/index.ts

```diff
@@ -463,7 +467,27 @@ export function defaultJSSetup() {
         setTimeout: undefined,
       }
       createContext(context)
-      return runInNewContext(js, context, { timeout: 1000 })
+
+      js = `
```

Comment:
If you are always overwriting the **js** argument, should you remove the parameters from the signature?

```ts
setJSRunner((js: string, context: Context) ...
```

becomes

```ts
setJSRunner((context: Context) ...
```